### PR TITLE
🐛add auth token and log for spec test 

### PR DESCRIPTION
**Pull request by OsmondJiang**

### CODE REVIEW

File: .deploy.yml [hunk 1]
```diff
@@ -28,7 +28,12 @@ jobs:
   condition: eq(variables['Build.Reason'], 'PullRequest')
   steps:
     - powershell: ./build.ps1
-
+    env:
+      OP_BUILD_USER_TOKEN: $(OpBuildUserToken)
```
Comment (yufeih) [inline .deploy.yml]: we don't need `OP_BUILD_USER_TOKEN` for build, it is to support running config migration script in impact testing

File: docs/specs/git.yml [hunk 2]
```diff
@@ -434,6 +434,7 @@ inputs:
     gitHub:
       userCache: user_profile.json
       resolveUsers: true
+      auth: {GITHUB_ACCESS_TOKEN}
```
Comment (superyyrrzz) [inline docs/specs/git.yml]: authToken

File: .deploy.yml [hunk 3]
```diff
@@ -56,14 +56,18 @@ jobs:
   - powershell: ./impact.ps1
     condition: eq(variables['Build.SourceBranch'], 'refs/heads/v3')
     env:
-      OP_BUILD_USER_TOKEN: $(OpBuildUserToken)
```
Comment (mingwli) [inline .deploy.yml]: this one is used for running config migration in edge-developer

Comment (OsmondJiang) [reply .deploy.yml]: @yufeih  is saying that you don't need it anymore?

**Pull request merged**